docs/api-reference/core/model.md

```diff
@@ -16,22 +16,94 @@ class.
 
 ## Usage
 
-Create model object by passing shaders, uniforms, and render it by passing updated uniforms.
+### Provide attribute data using Geometry object
+Create model object by passing shaders, uniforms, geometry and render it by passing updated uniforms.
+
```

Comment:
Check number of empty lines between various element (header, text, js) vs. other luma doc files.